model-optimizer/extensions/middle/L2NormToNorm.py

```diff
@@ -91,7 +112,7 @@ def replace_pattern(self, graph: Graph, match: dict):
                                                           np.ones(shape=int64_array([match['input'].shape[-1]]),
                                                                   dtype=match['input'].data_type),
                                                           {'name': output_name, 'eps': y,
-                                                           'across_spatial': 0, 'channel_shared': 0})
+                                                           'across_spatial': across_spatial, 'channel_shared': False})
         rename_node(normalize_node, output_name)
```

Comment:
Why all reduction axes are `1`? Why don't you specify the required attribute `eps_mode`? According the pattern, this attribute should be equal to `max`.

Reply:
If you want to specify only axes `1`, maybe you should write `int64_array([1])`?

Reply:
This value was leave "as is", because it doesn't used in related transformation `extensions/back/NormalizeToNormalizeL2.py`. The same for `eps_mode` attribute. We decide to make this quick fix firstly, and after that make some changes in IE and related changes in MO.